index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <img src="images.svg\image_geometry_1 (1).svg" alt="" class="top-left">
+        <nav><img src="images.svg\ic_menu (1).svg" alt="" class="hamburger"></nav>
+    </header>
```

Comment:
- [ ] The spacing around the hamburger image is not identical to that of the Figma template that you chose. Remember that one of the requirements is to make your design identical to that of the Figma template chosen. Kindly fix this. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/212402973-6bffeb82-1fc3-4bcd-90e3-8264954a5e72.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/212403049-86868853-399c-425b-8ada-2f25342fd73e.png)

Reply:
but that should not be a compulsory reason for to now pass my code i feel youre overscrutinizing here knowing i have limited codereviews also there is no given length its not something thats detectable we talking a difference of pixels

Reply:
@Gambit142

Reply:
@Gambit142

Reply:
Kindly try to replicate the Figma design as closely as possible, and yes, please follow the advice of the reviewer and don't keep the hamburger button glued to the top of your app as it makes your project look unprofessional.